.github/workflows/tacos_plan.yml

```diff
@@ -46,7 +46,6 @@ jobs:
       pull-requests: write
 
     name: list slices
-    if: github.event.pull_request.draft == false
```

Comment:
Do we want to add this check lower down to prevent draft PRs from acquiring the locks?